### ISSUE TITLE
ps-strategy-lowest-price FromTime ToTime 

### ISSUE THREAD

**Issue by Home-Control9000** [ottopaulsen/node-red-contrib-power-saver#167]
Hi,

In 'ps-strategy-lowest-price' module you can set 'FromTime'. This is a fixed value.
In practice for example you want to schedule the washmachine on the lowest hour price that day.
Is it possible to replace FromTime to **actual** time? This is usefull when you schedule in the morning for eaxmple 6am. The hours between 0 and 6am are not available anymore.  

Regards, Walter

**Comment by ottopaulsen** [comment]
Sorry, I do not understand the question. Are you looking for something like the [Fixed Schedule](https://powersaver.no/nodes/ps-strategy-fixed-schedule.html) node?

**Comment by Home-Control9000** [comment]
> Sorry, I do not understand the question. Are you looking for something like the [Fixed Schedule](https://powersaver.no/nodes/ps-strategy-fixed-schedule.html) node?

No, I want the lowest hour price. So i want to use the ps-strategy-lowest-price module. The problem is that the FromTime and ToTime are fixes values (hours) I want to have the lowest hourprice starting at  the **actual** hour. For example when i wakeup and want to schedule the washingmachine then the time is 6am. Then calculating the lowest price must start from that time. The hours before 6am are already  past.

Walter

**Comment by ottopaulsen** [comment]
Ok, so you want the lowest price from "now". You would have to use [dynamic config](https://powersaver.no/nodes/ps-strategy-lowest-price.html#dynamic-config). Find the current hour or maybe the next, and send it to the node as the new `fromTime`.

**Comment by Home-Control9000** [comment]
Thanks for your fast reply! I was looking for that:) I will try this. 
Someting else. Im connecting to the energyZero API (Netherlands ANWB Energie) Im already have a working Node-Red API for this. Im parsing every hour a JSON.
 I read i must create a specific JSON input but im just a beginner Node_red programmer. The JSON i received from Energyzero looks like this:

![image](https://github.com/ottopaulsen/node-red-contrib-power-saver/assets/18611208/e065ea4f-ee71-4d09-bae4-73ad30b019b3)

Can you give me some tips how i can convert this to the needed format as said in the docs? Thankyou.

![image](https://github.com/ottopaulsen/node-red-contrib-power-saver/assets/18611208/c3903b11-856e-457c-9e13-01ab9a0edfcc)
